client/src/partials/errors/404.html

```diff
@@ -0,0 +1,22 @@
+<div class="flex-content">
+  <div class="container-fluid" style="height: 100%;">
+    <div class="row">
+      <div class="col-md-4 col-md-offset-4">
+
+        <div class="panel panel-default content-error" data-error="404">
+          <div class="panel-heading clearfix text-center">
+            <strong>
+              <i class="fa fa-danger"></i>
+              <span class="text-danger">{{ "FORM.LABELS.ERROR_404" | translate }} </span>
```

Comment:
Given how similar these pages are I'm sure we could have one template and just insert different values. I know this is just a restructure of files and wasn't created in this pull request - just a potential future way to reduce maintenance cost.